kitsune/sumo/management/commands/merge.py

```diff
@@ -0,0 +1,57 @@
+from kitsune.settings import SUMO_LANGUAGES
+from django.core.management.base import BaseCommand
+from babel.messages.frontend import CommandLineInterface
```

Comment:
By Python custom, imports are typically ordered first by groups, with a single space between groups, and then alphabetically within each group, where the groups are:

- Python standard library imports (e.g., `os`)
- Third-party imports (e.g. `django`)
- Local imports (e.g., `kitsune.wiki.models)

and also I'd recommend importing `settings` from `django.conf`, and then using `settings.SUMO_LANGUAGES` instead of importing `SUMO_LANGUAGES` directly from `kitsune.settings`.

```suggestion
from babel.messages.frontend import CommandLineInterface
from django.conf import settings
from django.core.management.base import BaseCommand
```